src/routes/LandingPage.tsx

```diff
@@ -253,12 +251,7 @@ export default function LandingPage(): ReactElement {
     const publicationElement = project.publicationLink ? (
       <p>
         Related publication:{" "}
-        <a href={project.publicationLink.toString()} target="_blank" rel="noopener noreferrer">
-          {project.publicationName}
-          {/* Icon offset slightly to align with text */}
-          <FontAwesomeIcon icon={faUpRightFromSquare} size="sm" style={{ marginBottom: "-1px", marginLeft: "3px" }} />
-          <VisuallyHidden>(opens in new tab)</VisuallyHidden>
-        </a>
+        <ExternalLink href={project.publicationLink.toString()}>{project.publicationName}</ExternalLink>
```

Comment:
Misc. change I made while adding the GitHub documentation link in `GlossaryPanel`. This replaces the link with the shared `ExternalLink` component.